src/NewSubscription.m

```diff
@@ -202,6 +202,7 @@ -(IBAction)doSubscribe:(id)sender
 	// Validate the subscription, possibly replacing the feedURLString with a real one if
 	// it originally pointed to a web page.
 	rssFeedURL = [subscriptionModel verifiedFeedURLFromURL:rssFeedURL];
+	NSAssert(rssFeedURL != nil, @"No valid URL verified to attempt subscription !");
```

Comment:
I have not encountered that problem.
According to the doc, `NSAssert(condition, desc, ...)` will fail if `condition` evaluates to `NO`.
A similar use case here : https://github.com/ViennaRSS/vienna-rss/blob/master/src/ArticleController.m#L336